run/hello-broken/main.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START run_broken_service]
+
+// Sample hello demonstrates a difficult to troubleshoot service.
```

Comment:
More comment would be nice here. What is troublesome about it?

Reply:
The sample is used in a tutorial where the reader is eventually given this answer, but a comment in the first code snippet would be counter-productive. Is there somewhere else I might add this detail?